src/Domain/CoordinateMaps/CoordinateMap.hpp

```diff
@@ -471,7 +492,10 @@ CoordinateMap<SourceFrame, TargetFrame, Maps...>::inv_jacobian_impl(
           const auto& map_in_loop =
               std::get<(count != 0 ? count - 1 : 0)>(maps);
           if (LIKELY(not map_in_loop.is_identity())) {
-            mapped_point = map_in_loop(mapped_point);
+            CoordinateMap_detail::apply_map(
+                make_not_null(&mapped_point), map_in_loop, time, f_of_t_list,
```

Comment:
since `f_of_t_list` is not a list but an unordered map, would you mind renaming to say `functions_of_time`?